ros_pre-release.sh

```diff
@@ -71,9 +72,10 @@ travis_time_start run_prerelease
 travis_time_end  # run_prerelease
 
 travis_time_start show_testresult
-catkin_test_results --verbose && (echo 'ROS Prerelease Test went successful.'; exit 0) || error
+catkin_test_results --verbose && { echo 'ROS Prerelease Test went successful.'; RETVAL=0; success; } || { RETVAL=1; error; }
 travis_time_end  # show_testresult
```

Comment:
Good point, and my answer is no. `return` [in this line](https://github.com/ros-industrial/industrial_ci/pull/50/files#diff-69501095267bf26a6434210905896aa4R81) is important for the parent script to be able to capture the result. So we don't want this script to exit at this line yet and thus don't want to call `success`.

Hope this variation is not unnecessarily complicated, but maybe it is. I appreciate your opinion.